recipes/pro-mdnsd/all/conanfile.py

```diff
@@ -50,10 +52,22 @@ def layout(self):
     def source(self):
         get(self, **self.conan_data["sources"][self.version], strip_root=True)
 
+    def _get_log_level(self):
+        return {
+            "Fatal": "600",
+            "Error": "500",
+            "Warning": "400",
+            "Info": "300",
+            "Debug": "200",
+            "Trace": "100",
+            "PackageOption": "300"
```

Comment:
do we need this? check: https://github.com/conan-io/conan-center-index/pull/3786#discussion_r2044524111
```suggestion
```